src/wp-includes/class-wp-phpmailer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * PHPMailer extensions for WordPress
+ *
+ * @package WordPress
+ * @since 6.7.1
+ */
+
+/**
+ * PHPMailer child class
+ *
+ * Overrides the internationalization method in order to use WordPress' instead.
+ *
+ * @since 6.7.1
```

Comment:
The next major version is 6.8

```suggestion
 * @since 6.8.0
```